notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaAddressee.java

```diff
@@ -114,16 +115,19 @@ public void setName(String name) {
      */
     public void setRecipients(Set<JpaRecipient> recipients) {
         this.recipients.clear();
-        for (JpaRecipient recip : recipients) {
-            addRecipient(recip);
-        }
+        this.recipients.addAll(recipients);
     }
 
     /**
      * Adds the specified recipient to the current collection.
      */
     public void addRecipient(JpaRecipient recipient) {
         recipients.add(recipient);
-        recipient.setAddresseeId(getId());
     }
+
+    @Override
+    public String toString() {
+        return "JpaAddressee [id=" + id + ", name=" + name + ", type=" + type + ", recipients=" + recipients + "]";
```

Comment:
would [`org.apache.commons.lang3.builder.ToStringBuilder`](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ToStringBuilder.html) be a more effective way to generate this string?